### PR TITLE
Update kitchen config

### PR DESCRIPTION
**Pull request by armiller**

### CODE REVIEW

**Comment by tschuy** [comment]
Why are you removing the .kitchen.cloud.yml?

**Comment by armiller** [comment]
@tschuy this pr was made with a script. We are preparing all of the cookbooks for the new chef upgrades on the workstations. I talked about it at the summit if you were there. We will send out a staff email detailing the changes. I now realize we should have added some additional details in the pr message.

File: metadata.rb [hunk 1]
```diff
@@ -10,3 +10,8 @@
 depends 'gunicorn'
 depends 'python'
 depends 'supervisor'
+
+supports         'centos', '~> 6'
+supports         'centos', '~> 7'
+supports         'ubuntu', '14.04'
```
Comment (jordane) [inline metadata.rb]: Since when do we support ubuntu?

File: .kitchen.yml [hunk 2]
```diff
@@ -1,21 +1,17 @@
...
-
-platforms:
-  - name: centos-6.6
-    run_list:
-      - recipe[python-webapp-test::_centos]
-  - name: centos-7.0
-  - name: ubuntu-14.04
-  - name: ubuntu-12.04
-
+  attributes:
```
Comment (jordane) [inline .kitchen.yml]: This really belongs in the global kitchen config, imo.

File: .kitchen.yml [hunk 3]
```diff
@@ -1,21 +1,10 @@
...
-
 provisioner:
   name: chef_solo
   encrypted_data_bag_secret_key_path: test/integration/encrypted_data_bag_secret
   data_bags_path: test/integration/data_bags
 
-platforms:
-  - name: centos-6.6
-    run_list:
-      - recipe[python-webapp-test::_centos]
```
Comment (rettigs) [inline .kitchen.yml:13]: Not sure what to do about this.  Maybe move this functionality to the recipe and have it detect the platform?

Comment (armiller) [inline .kitchen.yml:13]: This can be moved to the `default` suite since that recipe checks for the platform version. I will update the pr.

**Comment by jldugger** [comment]
Do we even use python-webapp anymore?

```
$ knife search node "recipes:python.webapp"
0 items found```

**Comment by fahlmant** [comment]
@jldugger I don't think so

**Comment by jldugger** [comment]
@osuosl-cookbooks/chefs vote to close?

**Comment by ramereth** [comment]
This can be closed AFAIK

**Pull request closed without merging**